OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -347,10 +347,15 @@ bool CellIsEvacuating(Actor self, CPos cell)
 		}
 
 		public override void Cancel(Actor self, bool keepQueue = false)
+		{
+			Cancel(self, keepQueue, false);
+		}
+
+		public void Cancel(Actor self, bool keepQueue, bool forceStop)
```

Comment:
IMO `forceStop` is a bit misleading - if I didn't know better my first assumption is that this would force units to stop immediately even if between cells.

Can we please call this something like `forceClearPath` instead?